doc/ADMIN.md

```diff
@@ -12,6 +12,10 @@ Alternatively, you may open a 'Nextcloud shell' with `sudo yunohost app shell __
 
 ONLYOFFICE is an online rich text document editor which can be integrated in Nextcloud
 
+#### High Performance Backend
```

Comment:
Why this name ? And not notify push ? This name could mean other things like a more performant backend for the web ui…

Reply:
or push server

Reply:
"High performance backend" is just the way the Nextcloud team refers to it, cf https://help.nextcloud.com/t/nextcloud-hub-21-out-with-up-to-10x-better-performance-whiteboard-and-more-collaboration-features/108634